Cargo.toml

```
@@ -3,6 +3,6 @@ members = [
   "librad",
   "librad-test",
   "git-helpers",
-  "radicle-tracker",
-  "radicle-seed",
+  "tracker",
+  "seed",
```

```suggestion
  "radicle-tracker",
  "radicle-seed",
```

They are still referenced by the crate name in cargo, instead of the directory name.

---

Not really. If you apply that change, cargo will fail with: 

```
error: failed to read `/home/nuno/radicle-link/radicle-tracker/Cargo.toml`

Caused by:
  No such file or directory (os error 2)
```

---

:O My bad, it's the case for declaration deps. Disregard and sorry for the noise.

---

No worries mate :)